runtime-services-modeling/decisions-with-dmn.adoc

```diff
@@ -653,12 +661,91 @@ image::dmn-list-expression-example.png[]
 
 The following boxed list expression also identifies approved credit score agencies but uses FEEL logic to define the agency status (Inc., LLC, SA, GA) based on a DMN input node:
 
+[id="approved-credit-score-agency-list-example"]
 .Boxed list expression using FEEL logic for approved credit score agency status
 image::dmn-list-expression-example2.png[]
 
 .DMN diagram for approved credit score agency status
 image::dmn-list-expression-example2a.png[]
 
+[id="con-dmn-conditional-expressions"]
+==== Boxed conditional expressions
+
+[role="_abstract"]
+A boxed conditional offers a visual representation of an `if` statement using three rows.
+The expression in the `if` part must resolve to a boolean.
+When the `if` resolves to a `true`, `then` part is returned.
+When the `if` resolves to a `false`, `else` part is returned.
+
+.Conditional expression of MonthlyFee computation
+image::runtime-services-modeling/dmn-conditional-expression.png[]
+
+NOTE: This feature was introduced as part of {PRODUCT} 9.1 and DMN 1.5 releases.
+
+[id="con-dmn-for-expressions"]
+==== Boxed iterator expressions
+
+[role="_abstract"]
+A boxed iterator offers a visual representation of an iterator statement.
+For the `for` loop, the right part of the `for` displays the iterator variable name.
+The second row holds an expression representing the collection that will be iterated over.
```

Comment:
```suggestion
The second row must contain an expression that resolves to a collection, which will be iterated over.
```

Reply:
Or 
The `in` row must contain an expression that resolves to a collection, which will be iterated over. 
Just personal preference here, but at first glance it sounded like the second row and the in row were different rows.